src/app/attendance/actions/updateAttendance.ts

```diff
@@ -0,0 +1,25 @@
+"use server";
+
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { AttendanceFormValues } from "../components/AttendanceFormBuilder";
+import { redirect } from "next/navigation";
+import { Routes } from "@/lib/routes";
+
+export const updateAttendance = async (
+  id: number,
+  { startTime, endTime, memberId }: AttendanceFormValues
```

Comment:
@ghulamhafeez  we can pass one prop instead of these

Reply:
Fixed